### PR TITLE
tor: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by tricantivu** [comment]
Overall, LGTM. However, I left a comment with a question regarding command examples.

File: pages.es/linux/tor.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- Comprueba el estado de Tor:
+
+`tor --status`
+
+- Ejecuta solo como cliente:
+
+`tor --client`
+
+- Ejecuta como relé:
```
[the remark this answers is not in the document]
Comment (kbdharun) [reply pages.es/linux/tor.md:26]: > Are duplicate command examples allowed? If not, one of the two examples should be deleted in both pages.

We generally prefer not having duplicate examples, but we do allow them when the duplicate has added flags or options. But in this case there seems to be no difference, so IG we can remove one example from here and in the English page.

**Review by tricantivu — APPROVED** [review]

File: pages.es/linux/tor.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tricantivu) [reply pages.es/linux/tor.md:26]: Got it! Thank you both.

[the remark this answers is not in the document]
Comment (spageektti) [reply pages.es/linux/tor.md:26]: #12855

[the remark this answers is not in the document]
Comment (kant) [reply pages.es/linux/tor.md:22]: @spageektti updated

**Review by spageektti — APPROVED** [review]

**Pull request merged**